/*
*  This file is part of aasdk library project.
*  Copyright (C) 2018 f1x.studio (Michal Szwaj)
*
*  aasdk is free software: you can redistribute it and/or modify
*  it under the terms of the GNU General Public License as published by
*  the Free Software Foundation; either version 3 of the License, or
*  (at your option) any later version.

*  aasdk is distributed in the hope that it will be useful,
*  but WITHOUT ANY WARRANTY; without even the implied warranty of
*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
*  GNU General Public License for more details.
*
*  You should have received a copy of the GNU General Public License
*  along with aasdk. If not, see <http://www.gnu.org/licenses/>.
*/

syntax="proto2";

package oaa.proto.enums;

message ShutdownReason
{
    enum Enum
    {
        NONE = 0;
        USER_SELECTION = 1;
        DEVICE_SWITCH = 2;
        NOT_SUPPORTED = 3;
        NOT_CURRENTLY_SUPPORTED = 4;
        PROBE_SUPPORTED = 5;
        WIRELESS_PROJECTION_DISABLED = 6;
        POWER_DOWN = 7;
        USER_PROFILE_SWITCH = 8;
    }
}
